tools/xcodeproj_shims/installers/_indexstore.sh

```diff
@@ -30,9 +30,13 @@ readonly xcode_module="$BUILT_PRODUCTS_DIR/\$1.swiftmodule/$ARCHS.swiftmodule"
 readonly bazel_external="$bazel_root/external"
 readonly xcode_external="$BAZEL_WORKSPACE_ROOT/bazel-$(basename "$SRCROOT")/external"
 
+readonly remote_developer_dir="^/.*/.+?\.app/Contents/Developer"
```

Comment:
@thii  this effectively maps _any_ of the Xcodes to the local one? It seems like a good thing to have. Maybe add a comment on this. I think it's fine to assume it's always `Contents/Developer` is the developer dir for now